service/src/main/java/uk/gov/hmcts/reform/fpl/model/RespondentParty.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
+import uk.gov.hmcts.reform.fpl.model.common.Party;
+import uk.gov.hmcts.reform.fpl.model.common.TelephoneNumber;
+
+import java.util.Date;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
I wonder if this is needed here. It might be enough to annotate only common types such as Telephone or EmailAddress.